docs/index.rst

```diff
@@ -3,7 +3,7 @@ Qiskit |version| documentation
 ##############################
 
 Qiskit is open-source software for working with quantum computers
-at the level of circuits, pulses, and algorithms.
+at the level of circuits and pulses.
```

Comment:
😢

Reply:
The "fix" was merged 10 mins ago https://github.com/Qiskit/qiskit-neko/pull/34. I've re-triggered the CI to see if it passes now.